installer/scripts/omsadmin.sh

```diff
@@ -325,6 +326,35 @@ set_proxy_setting()
     fi
 }
 
+is_scom_port_open()
+{
+    OIFS=$IFS
+# Ignore formatting. It was required for IFS to take new line on both bash and sh
+IFS='
+'
+    for line in `cat $OMI_CONF_FILE | grep httpsport`
+    do
+        echo $line | grep -E '^#.*$' > /dev/null 2> /dev/null
```

Comment:
For all the spots where you forward to `/dev/null`, please use this syntax: `> /dev/null 2>&1`. We use a couple different formats for this throughout the code right now, but I'm trying to standardize it.